referrer-policy/no-referrer-policy/no-referrer-policy.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Check that sub-resource gets the referrer URL when no explicit
+           Referrer Policy is set.</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <!-- Common global functions for referrer-policy tests. -->
+    <script src="../generic/common.js"></script>
+  </head>
+  <body>
+    <h1>Check that sub-resource gets the referrer URL when no explicit Referrer
+        Policy is set.</h1>
+
+    <p>This page loads sub-resources (iframes) and waits for messages containing
+       referrer URLs from children.</p>
+
+    <script>
+      var documentUrl = document.location.toString();
+
+      // The referrer URL should be this document's URL adjusted according to
+      // this section:
+      // https://w3c.github.io/webappsec/specs/referrer-policy/#strip-url
+      var expectedReferrerUrl = stripUrlForUseAsReferrer(documentUrl);
+
+      // Path to the sub-resource page which feeds back received referrer URL.
+      var subresourceUrlPath = "/referrer-policy/no-referrer-policy" +
+                               "/no-referrer-policy.subresource.py";
```

Comment:
This is already reused in the tests below. Once I add more scenarios I'll rethink how to separate them and reuse a bunch of stuff.